<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="pageAuth">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="tab_numLabel">
              <property name="text">
               <string>personnel number</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="tab_numLineEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="passwordLabel">
              <property name="text">
               <string>password</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="passwordLineEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonAuth">
          <property name="text">
           <string>Авторизоваться</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageCashierMenu">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButtonSnowSeances">
          <property name="minimumSize">
           <size>
            <width>135</width>
            <height>25</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>135</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Show Avaible Seances (Sell/Return Tickets)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonCloseDb">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Close Database</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageAdmiMenu">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="_2">
          <item>
           <layout class="QVBoxLayout" name="filmMenu">
            <item>
             <widget class="QPushButton" name="pushButtonSelectFilms">
              <property name="text">
               <string>Select/Update films</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonAddFilm">
              <property name="text">
               <string>Add film</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonDeleteFilms">
              <property name="text">
               <string>Delete film</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="seanceMenu">
            <item>
             <widget class="QPushButton" name="pushButtonSelectSeance">
              <property name="text">
               <string>Select/Update seances</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonAddSeance">
              <property name="text">
               <string>Add seance</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonDeleteSeance">
              <property name="text">
               <string>Delete seance</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="userMenu">
            <item>
             <widget class="QPushButton" name="pushButtonAddUser">
              <property name="text">
               <string>Add user</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonCloseDb_2">
          <property name="text">
           <string>Close Database</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageAddUser">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="labelPersonnelNumber">
            <property name="text">
             <string>Personnel Number*</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditPersonnelNumber"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelFIO">
            <property name="text">
             <string>FIO*</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEditFIO"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Position*</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>cashier</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>admin</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelEducation">
            <property name="text">
             <string>Education</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEditEducation"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="labelTelephone">
            <property name="text">
             <string>Telephone*</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="lineEditTelephone"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="labelBirthDate">
            <property name="text">
             <string>Birth Date</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QDateEdit" name="dateEdit">
            <property name="displayFormat">
             <string>dd.MM.yyyy</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelPassword">
            <property name="text">
             <string>Password*</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditPassword"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonAddUser_2">
          <property name="text">
           <string>Add User</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>475</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
